covid-nlp/main.ipynb

```diff
@@ -0,0 +1,531 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">&gt;&gt;&gt; matches_with_conlicts = session.query("...")
    [
        ("my patient doesnt have covid","not covid",1),
        ("my patient doesnt have covid","covid",2),
        ("my patient has covid","covid",2),
    ]




&gt;&gt;&gt; matches_without_conflict = matches_with_conlicts.group_by("document").agg({'penalty':take_min})
    [
        ("my patient doesnt have covid","not covid",1),
        ("my patient has covid","covid",2),
    ]



session.import_csv(matches_without_conflict,'matches_without_conflict')
</pre><p>This is for solving conflicts, then you can rewrite to a new text regularly</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/spanner_workbench/pull/114/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>